app/views/responsibilities/confirm.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :switcher do %>
+  <%= render '/layouts/prison_switcher' %>
+<% end %>
+
+<%= form_for(@responsibility, url: prison_responsibilities_path(@prison, @responsibility)) do |f| %>
+  <%= f.hidden_field(:nomis_offender_id) %>
+  <%= f.hidden_field(:reason) %>
+  <%= f.hidden_field(:reason_text) %>
+
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset" aria-describedby="how-contacted-conditional-hint">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+        <h1 class="govuk-fieldset__heading">
+          Confirm change of responsibility for this case
+        </h1>
+      </legend>
+    </fieldset>
+    <p class="govuk-body">
+      The community probation team will be responsible for this case
+    </p>
+    <p class="govuk-body">
+      We will email the community probation team at <%= @ldu_email_address %>.
```

Comment:
Was there any discussion on what should happen if no LDU email is available?  Presumably something like giving the LDU name, and then suggesting they contact them?

Reply:
no you're right - I'm unsure what is supposed to happen here.

Reply:
I think @lmoney-moj will be able to help with content for that case.